### DISCUSSION TITLE
FR problème avec ma switch

### DISCUSSION THREAD

**Discussion by arknasa** [community/community#142966] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Ma switch à été craqué par une personne que je ne peut plus contacter aujourd'hui, j'avais appris à remettre le crack (qui part quand la swich s'éteint) et à mettre des jeux dessus (sans problème)
Récement j'ai refait la même avec celle de ma soeur qui n'as eu aucun problème
Ma switch était éteinte, je l'ai chargé à fond
je l'ai éteinte
redémarrer en mode RCM (mon ordi à bien vue qu'elle était comme ça)
j'ai réinjecter le playload : hekate_ctcaer_6.0.7.bin
ils dit que tout est bon
mais l'écran de ma siwtch reste noir (normalement y'a un truc qui s'affiche puis je doit aller dans des fichier et mettre en route mais comme tout est noir je peut rien faire)
je doit appuyer longtemps sur la switch (comme si je devait l'éteindre de force) puis le refaire pour la rallumer en mode normal (donc pas de crack)
Si vous pouvez m'aider ça serais trop cool Merci d'avance

**Comment by ebndev** [comment]
Hi @arknasa, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
